### DISCUSSION TITLE
我太爱这个插件的pdf翻译功能了，请问未来可以尝试推出zotero的翻译插件么！？

### DISCUSSION THREAD

**Discussion by YuDi0-0** [immersive-translate/immersive-translate#1267] [Ideas]
### 新功能 | New Feature

希望可以推出zotero的翻译插件，用于论文阅读

### 为什么您想要添加这些功能 | Why you want to add these features

1.zotero是目前新兴的极为强大的开源论文管理软件，市场前景巨大；本插件的pdf翻译功能是科研人刚性需求
2.阅读论文时需要大量的翻译工作，目前zotero软件中的翻译插件还不够完善，大部分用户还在使用跳转知云、小绿鲸等软件的方式
3.大量用户能够接受知识付费功能，但前提是插件功能足够完善
4.个人观点 ，目前插件的定价过高，一年700的价格如果没有单位报销用户很难自己消化。。。。建议降低价格薄利多销，并期望增加更多引人入胜的功能。

**Comment by theowenyoung** [comment]
感谢建议~ 但是目前团队的精力还没法投入到一个新的产品上～

>  4.个人观点 ，目前插件的定价过高，一年700的价格如果没有单位报销用户很难自己消化。。。。建议降低价格薄利多销，并期望增加更多引人入胜的功能。

沉浸式翻译实际上是完全免费的插件，目前只对有需求的高频用户提供了一个可选的第三方翻译额度付费套餐，同时用户亦可以使用自定义的 Key.

未来会考虑提供一个轻量的 lite 套餐供低频用户选择。

**Reply by YuDi0-0** [reply]
本心是看到了这么用心的团队做了个很人性化的产品 想支持一下的，打开pro看了一下价格没顶住哈哈哈哈，如果可以的话，希望日后出一个永久会员，年费的还是会让用户有后顾之忧，总算着一天多少钱。

**Reply by YuDi0-0** [reply]
很理解团队没有额外的精力，但是依然希望未来有一天能够推出一款用于zotero的插件。
https://github.com/windingwind/zotero-pdf-translate
这里把现在的一个zotero翻译插件的hub地址贴在这里，相信其中有很多类似，希望未来有时间的时候团队可以参考完善。真的迫切需要一款好用的翻译插件用于论文阅读，殷切希望有一天可以用上zotero中的沉浸式翻译。
              祝好

**Reply by moolawooda** [reply]
zotero貌似采用了firefox API来支持[插件](https://www.zotero.org/support/dev/client_coding/plugin_development)，甚至他的插件跟firefox的一样都是.xpi格式的，不知道是否能减少移植的工作量

**Comment by liruifengv** [comment]
您好，我们的 Zotero 插件发布了，目前 0.0.1 版本，欢迎下载使用一下，在 [Release](https://github.com/immersive-translate/zotero-immersivetranslate/releases) 下载 xpi 安装

https://github.com/immersive-translate/zotero-immersivetranslate
